### PR TITLE
updated the validation schema and translations based on 84583

### PR DESCRIPTION
**Pull request by alexunique0519**
### Description

Updated the validation schema and messages translations according to 84583. For details, please refer to task descriptions. 

### What to test for/How to test

Pull changes and run

### CODE REVIEW

File: utils/api/definitions/schemas.ts [hunk 1]
```diff
@@ -61,35 +61,43 @@ export const RequestSchema = Joi.object({
...
-          'any.unknown': ValidationErrors.yearsInCanadaNotEnough,
-        }),
-      }),
+    .when('livingCountry', {
+      is: Joi.string().valid('CAN'),
+      then: Joi.when(
+        'yearsInCanadaSince18',
+        //9.9 to disable 10,since the input only accepts integer, this is good
+        {
+          is: Joi.number().max(9.9),
```
Comment (alex-solo) [inline utils/api/definitions/schemas.ts]: Is it possible to use 10 and 20 in validating the input and not 9.9 and 19.9?

File: i18n/web/en.ts [hunk 2]
```diff
@@ -156,8 +156,11 @@ const en: WebTranslations = {
     [ValidationErrors.partnerAgeOver150]:
       "Your partner's age should be less than 150.",
     [ValidationErrors.oasAge65to70]: 'You must enter an age between 65 and 70.',
-    [ValidationErrors.yearsInCanadaNotEnough]:
-      'The number of years you have lived in Canada is not enough to receive any old age benefits.',
+    [ValidationErrors.yearsInCanadaNotEnough10]:
+      //'The number of years you have lived in Canada is not enough to receive any old age benefits.',
```
Comment (alex-solo) [inline i18n/web/en.ts]: Why are some of these commented out?

**Review by alex-solo — CHANGES_REQUESTED** [review]
See comments

File: utils/api/definitions/schemas.ts (hunk 1 above)
Comment (alexunique0519) [reply utils/api/definitions/schemas.ts]: if we use max(10) here, it mean [0, 10], which includes 10 years. But we only need to show the error when user lives under 10 years. So we can't use max(10)

File: i18n/web/en.ts (hunk 2 above)
Comment (alexunique0519) [reply i18n/web/en.ts]: The error messages have been changed, I will remove the commented messages.

File: utils/api/definitions/schemas.ts (hunk 1 above)
Comment (alex-solo) [reply utils/api/definitions/schemas.ts]: I mean using a different Joi method (something like "under" or "lessThan" if those exist) to work in terms of numbers that we care about (10 and 20)

Comment (alexunique0519) [reply utils/api/definitions/schemas.ts]: replace the max() with less(), works like a charm

**Review by alex-solo — APPROVED** [review]
issues resolved

**Pull request merged**